WooCommerce/WCObject.cs

```diff
@@ -347,11 +347,11 @@ public WCObject(RestAPI api)
 
         #region "webhooks..."
 
-        public async Task<string> GetWebhooks(Dictionary<string,string> parms = null)
+        public async Task<WebhookList> GetWebhooks(Dictionary<string,string> parms = null)
         {
             string json = await API.SendHttpClientRequest("webhooks", RequestMethod.GET, string.Empty, parms);
             json = json.Substring(json.IndexOf('['), json.Length - json.IndexOf('[') - 1);

```

Comment:
The below code will work for both cases.
`json = json.Substring(json.IndexOf(':') + 1, json.Length - json.IndexOf(':') - 2);`